api/src/pcapi/routes/backoffice/offerers/offerer_blueprint.py

```diff
@@ -169,6 +169,9 @@ def _render_offerer_details(offerer_id: int, edit_offerer_form: offerer_forms.Ed
         delete_offerer_form=empty_forms.EmptyForm(),
         show_subscription_tab=show_subscription_tab,
         active_tab=request.args.get("active_tab", "history"),
+        zendesk_sell_synchronisation_form=empty_forms.EmptyForm()
+        if utils.has_current_user_permission(perm_models.Permissions.MANAGE_PRO_ENTITY)
```

Comment:
Ici aussi, ne pas afficher le bouton si on sait déjà qu'il ne pourra rien faire : 
```suggestion
        if not zendesk_sell.is_offerer_only_virtual(offerer) and utils.has_current_user_permission(perm_models.Permissions.MANAGE_PRO_ENTITY)
```
Cela nécessite peut-être un `joinedload` supplémentaire pour charger les `isVirtual` des `managedVenues` dans `_load_offerer_data`.